src/apps/patron-page/PatronPage.dev.tsx

```diff
@@ -240,11 +240,20 @@ export default {
       defaultValue: "You used @this ebooks out of you quota of @that ebooks",
       control: { type: "text" }
     },
-    phoneInputMessageText: {
+    patronPagePhoneInputMessageText: {
```

Comment:
If you have refactored this data prop, what about dpl-cms then?

Reply:
it is fixed here: https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/427/files